isso/utils/html.py

```diff
@@ -54,6 +54,9 @@ def set_links(attrs, new=False):
 def Markdown(extensions=("strikethrough", "superscript", "autolink",
                          "fenced-code"), flags=[]):
 
+    # Normalize render extensions for misaka 2.0
+    extensions = list(map(lambda x: x.replace("_", "-"), extensions))
```

Comment:
List comprehension would be prettier I guess?

Reply:
Yes, definitely:

```python
extensions = [x.replace("_", "-") for x in extensions]
```

Reply:
Yup.